src/utils/validators/isValidValue.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * A valid value it is consider if it is:
+ * value == true
+ * length > 0
+ * */
+
+export const isValidValue = value =>
```

Comment:
Add unit tests for this and `isValidUrl`

Reply:
I added some test like these:

```
test('undefined is not valid value', () => {
    expect(isValidValue(undefined)).toBeFalsy()
})
```

```
test('"https://debug.dhis2.org" is a valid URL', () => {
    const url = 'https://debug.dhis2.org'
    expect(isValidURL(url)).toBeTruthy()
})
```

Reply:
That's a good start. Could be expanded later.